sdk/schemaregistry/Azure.Data.SchemaRegistry/tests/SchemaFormatTests.cs

```diff
@@ -12,15 +13,75 @@ public void VerifyAvroFormat()
         {
             Assert.AreEqual("Avro", SchemaFormat.Avro.ToString());
         }
+
         [Test]
         public void VerifyJsonFormat()
         {
             Assert.AreEqual("JSON", SchemaFormat.Json.ToString());
         }
+
         [Test]
         public void VerifyCustomFormat()
         {
             Assert.AreEqual("Custom", SchemaFormat.Custom.ToString());
         }
+
+        [Test]
+        public void VerifyProtobufFormat()
+        {
+            Assert.AreEqual("Protobuf", SchemaFormat.Protobuf.ToString());
+        }
+
+        [Test]
+        public void VerifyAvroToContentType()
+        {
+            Assert.AreEqual(ContentType.Avro, SchemaFormat.Avro.ToContentType());
+        }
+
+        [Test]
+        public void VerifyJsonToContentType()
+        {
+            Assert.AreEqual(ContentType.Json, SchemaFormat.Json.ToContentType());
+        }
+
+        [Test]
+        public void VerifyCustomToContentType()
+        {
+            Assert.AreEqual(ContentType.Custom, SchemaFormat.Custom.ToContentType());
+        }
+
+        [Test]
+        public void VerifyProtobufToContentType()
+        {
+            Assert.AreEqual(ContentType.Protobuf, SchemaFormat.Protobuf.ToContentType());
+        }
+
+        [Test]
+        public void VerifyAvroFromContentType()
+        {
+            var fromContentType = SchemaFormat.FromContentType("application/json; serialization=Avro");
+            Assert.AreEqual(SchemaFormat.Avro, fromContentType);
+        }
+
+        [Test]
+        public void VerifyJsonFromContentType()
+        {
+            var fromContentType = SchemaFormat.FromContentType("application/json; serialization=Json");
```

Comment:
When writing this test I realized that the version returned in the ContentType header by the service is actually:
`"application/json; serialization=Json"`
while the ContentType.JsonValue generated by the swagger is set to:
`"application/json; serialization=json"`
Is this something that needs to be fixed in the swagger? Or should we use a transform to generate it correctly?

Reply:
We should, and they should fix either the swagger or the service.